.github/workflows/ci.yml

```
@@ -0,0 +1,23 @@
+name: test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [8, 11, 17-ea]
+        scala: [2.13.6, 3.0.1]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

We need to use

```suggestion
    - uses: actions/checkout@v2
      with:
        fetch-depth: 0
```

To get the Git tags information so that `dynver` and `version-policy` can do their job.

---

good catch